cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -215,6 +228,15 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
           Praos.babbageMaxTxCapacityOverrides =
             TxLimits.mkOverrides TxLimits.noOverridesMeasure
         }
+        Praos.ProtocolParamsConway {
+          -- This is /not/ the Babbage protocol version. It is the protocol
+          -- version that this node will declare that it understands, when it
+          -- is in the Babbage era. Since Babbage is currently the last known
+          -- protocol version then this is also the Babbage protocol version.
+          Praos.conwayProtVer = ProtVer 9 0,
```

Comment:
Shouldn't we be hiding this behind the `TestEnableDevelopmentHardForkEras` flag: https://github.com/input-output-hk/cardano-node/commit/e4240315820ce55ec70e718610ed63e7e380fc84#diff-8c649864fe4b96d142cfd4c2bb9e32a9510fe737c6ad4e13e04b585ee15a6b74R210